ports/stm32/mboot/main.c

```diff
@@ -1574,6 +1553,28 @@ void stm32_main(int initial_r0) {
     }
 }
 
+void led_status() {
+    static uint32_t ss = 0;
```

Comment:
How does this function behave when get_reset_mode() is executing?

Reply:
hmm.. probably poorly. I don't see get_reset_mode on my board sorry, hadn't thought of it. The systick handling probably needs a mask case for when other code is needing to manage the leds.

Reply:
maybe signal it with a special dfu_state?